package.json

```diff
@@ -21,6 +21,7 @@
   },
   "scripts": {
     "prepublish": "./node_modules/coffee-script/bin/coffee -o lib/ src/",
+    "postinstall": "./node_modules/coffee-script/bin/coffee -o lib/ src/",
```

Comment:
coffee is a devDependency, so postinstall won't work - is there any specific reason you added it?